rightmesh/contracts/MeshToken.sol

```diff
@@ -41,15 +38,12 @@ contract MeshToken is CappedToken, PausableToken {
   }
 
   /**
-   * @dev overriding Pausable#pause method to only allow pausing the contract once
-   * Once unpaused token can not be paused again
+   * @dev overriding Pausable#pause method to do nothing
+   * Paused is set to true in the constructor itself, making the token non-transferrable on deploy.
+   * once unpaused the contract cannot be paused again.
    * adding this to limit owner's ability to pause the token in future.
    */
-  function pause() onlyOwner whenNotPaused public {
-    require(pausedOnce == false);
-    pausedOnce = true;
-    super.pause();
-  }
+  function pause() onlyOwner whenNotPaused public {}
```

Comment:
I wish that solidity had an override keyword that would break compilation if nothing was overriden (from a typo in the function name for example).